memory/src/main/java/org/opensearch/ml/memory/index/ConversationMetaIndex.java

```diff
@@ -87,7 +87,7 @@ public void initConversationMetaIndexIfAbsent(ActionListener<Boolean> listener)
             log.debug("No conversational meta index found. Adding it");
             CreateIndexRequest request = Requests
                 .createIndexRequest(META_INDEX_NAME)
-                .mapping(ConversationalIndexConstants.META_MAPPING)
+                .mapping("{\"_doc\":" + ConversationalIndexConstants.META_MAPPING + "}")
```

Comment:
FYI after further research, another possible (and less controversial) fix is using a differnet method that includes the MediaType

```suggestion
                .mapping(ConversationalIndexConstants.META_MAPPING, XContentType.JSON)
```

Reply:
done